src/main/resources/db/migration/0350/V381__PublishedProjectDetails.sql

```diff
@@ -36,6 +35,7 @@ CREATE TABLE funder.published_project_details (
     project_highlight_photo_value_id BIGINT REFERENCES docprod.variable_values,
     project_zone_figure_value_id BIGINT REFERENCES docprod.variable_values,
     standard TEXT,
+    tf_reforestable_land NUMERIC,
```

Comment:
Editing an existing migration will cause the server to bomb out when it starts up because the migration file's checksum won't match the checksum of the version that was already applied to the database. This change would need to be made in the previous PR before merging it.

Reply:
Ah whoops, I made that change in the same file before creating the other PR, but must have already staged the changes and forgotten to restage this file. Will fix